src/main/java/edu/harvard/iq/dataverse/api/Access.java

```diff
@@ -107,6 +107,9 @@
 import javax.ws.rs.core.Response;
 import static javax.ws.rs.core.Response.Status.BAD_REQUEST;
 import javax.ws.rs.core.StreamingOutput;
+import static edu.harvard.iq.dataverse.util.json.JsonPrinter.json;
```

Comment:
Remove these as well?

Reply:
oh, I definitely didn't add this extra line on purpose; must be a side effect of git's auto-merge, when I was syncing up with develop...